### ISSUE TITLE
Should fix-vf-meta set a flag value for the "regular" values to set `ELIDABLE_AXIS_VALUE_NAME`?

### ISSUE THREAD

**Issue by thundernixon** [googlefonts/gftools#111]
As I understand it, the `Flags` value of and entry in the `AxisValue` fields of the `STAT` table specify which instances have names that can be left out of composed font names. 

I'm unclear on this, but based on what I can tell from the (MS OT STAT Flags spec](https://docs.microsoft.com/en-us/typography/opentype/spec/stat#flags), it seems that for an `AxisValue` like "Regular," the `flags` should be set to `2` rather than `0`, as a current process of fontmake + fix-vf-meta will result in.

The spec says of the `ELIDABLE_AXIS_VALUE_NAME` value:

> If set, it indicates that the axis value represents the “normal” value for the axis and may be omitted when composing name strings.

This is different from the `elidedFallbackNameID`, which I didn't initially realize. Whereas the `2` of the `elidedFallbackNameID` refers to nameID 2, the `2` in the `Flags` value would just state an axis value can be left out of composed names. I'm not sure how vital this is, however, as MS Word already does leave the "Regular" name off of composed font names.

Here's a current Regular entry:

```
<AxisValue index="1" Format="1">
    <AxisIndex value="0"/>
    <Flags value="0"/>
    <!-- Regular -->
    <ValueNameID value="259"/>
    <Value value="400.0"/>
</AxisValue>
```

Here's what I suspect it should be (line 3 changed):

```
<AxisValue index="1" Format="1">
    <AxisIndex value="0"/>
    <Flags value="2"/>
    <!-- Regular -->
    <ValueNameID value="259"/>
    <Value value="400.0"/>
</AxisValue>
```

I'm posting this because I've been looking into this value, and I _think_ it could be a spot where we could slightly improve the meta fixer. However, I have uncertainty on whether I'm reading the spec in a completely correct way, so if I am misunderstanding the `Flags` entry, we shouldn't change this.

**Comment by thundernixon** [comment]
Here's a particular example that seems to indicate that Flags should be set as I suggested above:

![image](https://user-images.githubusercontent.com/7355414/51407638-76f73680-1b2a-11e9-8784-748c2165b40c.png)

**Comment by m4rc1e** [comment]
Good find. I'll look into this on Monday.

**Comment by davelab6** [comment]
Great work Stephen!!

**Comment by m4rc1e** [comment]
This has been fixed in #237
